packages/addons/service/dispmanx_vnc/source/system.d/service.system.dispmanx_vnc.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=dispmanx_vnc
+After=kodi.service
+
+[Service]
+ExecStart=/bin/sh /storage/.kodi/addons/service.system.dispmanx_vnc/bin/dispmanx_vncserver-service
```

Comment:
From a purely naming perspective, calling the wrapper shell script a service is rather confusing - why not just call it `dispmanx_vncserver.sh`?